frames/frames.go

```diff
@@ -35,4 +35,7 @@ var FrameMap = map[string]FrameType{
 	"rick":            Rick,
 	"can-you-hear-me": Rick,
 	"donut":           Donut,
+	"spidyswing":      spidy
+	"batman-running":  BNR
+	"BNR":             BNR
```

Comment:
This won't compile, you'll need to add trailing commas to each of these lines:
```suggestion
	"spidyswing":      spidy,
	"batman-running":  BNR,
	"BNR":             BNR,
```